websites/A/AniList/dist/metadata.json

```diff
@@ -22,7 +23,7 @@
 		"vi_VN": "AniList: Theo dõi, tìm kiếm, chia sẻ Anime & Manga - Đăng ký ngay hôm nay!"
 	},
 	"url": "anilist.co",
-	"version": "2.0.1",
+	"version": "2.1.0",
```

Comment:
```suggestion
	"version": "2.0.2",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8